404.html

```diff
@@ -5,6 +5,96 @@
   <script type="text/javascript">
 	window.isErrorPage = true;
 	window.errorCode = '404';
+
+	const redirectMap = new Map(
+		[
+			['ae_ar', '/'],
+			['ae_en', '/'],
+			['africa', '/'],
+			['ar', '/es/'],
+			['at', '/de/'],
+			['au','/'],
+			['be_en', '/'],
+			['be_fr/', '/fr/'],
+			['be_nl', '/nl/'],
+			['bg', '/'],
+			['ca', '/'],
+			['ca_fr', '/fr/'],
+			['ch_de', '/de/'],
+			['ch_fr', '/fr/'],
+			['ch_it', '/it/'],
+			['cis_en', '/'],
+			['cis_ru', '/'],
+			['cl', '/es/'],
+			['co', '/es/'],
+			['cy_en', '/'],
+			['cz', '/'],
+			['ee', '/'],
+			['gr_en', '/'],
+			['hk_en', '/'],
+			['hk_zh', '/tw/'],
+			['hu', '/'],
+			['id_en', '/'],
+			['id_id', '/'],
+			['ie', '/'],
+			['il_en', '/'],
+			['il_he', '/'],
+			['in', '/'],
+			['in_hi', '/'],
+			['la', '/es/'],
+			['lt', '/'],
+			['lu_de', '/de/'],
+			['lu_en', '/'],
+			['lu_fr', '/fr/'],
+			['lv', '/'],
+			['mena_ar', '/'],
+			['mena_en', '/'],
+			['mt', '/'],
+			['mx', '/es/'],
+			['my_en', '/'],
+			['my_ms', '/'],
+			['nz', '/'],
+			['pe', '/es/'],
+			['ph_en', '/'],
+			['ph_fil', '/'],
+			['pl', '/'],
+			['pt', '/br/'],
+			['ro', '/'],
+			['ru', '/'],
+			['sa_ar', '/'],
+			['sa_en', '/'],
+			['sea', '/'],
+			['sg', '/'],
+			['si', '/'],
+			['sk', '/'],
+			['th_en', '/'],
+			['th_th', '/'],
+			['tr', '/'],
+			['ua', '/'],
+			['uk', '/'],
+			['vn_en', '/'],
+			['vn_vi', '/'],
+		]	
+	);
+	
+	(function (win, doc) {
+		const { host, hostname, href, origin, pathname, port, protocol} = win.location;	
+
+		let originalPath = pathname.split("/");
+		let userLocale = originalPath[1]; 
+		let redirectMapping = redirectMap.get(userLocale);
+		
+		if (redirectMapping){
+			let localeCookie = userLocale;
+			// making sure just the specific string between the forward slashes are replaced.
+			userLocale = "/"+ userLocale +"/";
+			let redirectURL = href.replace(userLocale, redirectMapping);
+			doc.cookie = " international = "+ localeCookie + ";" + "Path = /; domain = adobe.com";
```

Comment:
looks like the domain isn't set correctly.. I've tried with https://client-redirect--express-website--adobe.hlx.page/in/express/pricing, its taking to https://client-redirect--express-website--adobe.hlx.page/express/pricing but the `international` cookie isn't updated. Can you try using `domain=.adobe.com` ?

Reply:
may be you can re-write this simply like 
```
` international = ${localeCookie}; Path = /; domain = adobe.com;`
```

Reply:
@ravkiran I have changed it to your more elegant version..Resolved